test/integration/02-user-wallet/02-index.spec.ts

```diff
@@ -134,9 +135,11 @@ describe("UserWallet", () => {
 
   describe("getStringCsv", () => {
     const csvHeader =
-      "voided,approved,_id,accounts,credit,debit,_journal,book,unix,date,datetime,currency,username,type,hash,txid,fee,feeUsd,sats,usd,memo,memoPayer,meta,pending"
+      "id,walletId,type,credit,debit,fee,currency,timestamp,pendingConfirmation,journalId,lnMemo,usd,feeUsd,recipientWalletId,username,memoFromPayer,paymentHash,pubkey,feeKnownInAdvance,address,txHash"
     it("exports to csv", async () => {
-      const base64Data = await userWallet0.getStringCsv()
```

Comment:
Would we be able to make this switch in `graphql-old-server.ts` as well and delete this `getStringCsv` method entirely?

Reply:
I'm not actively looking at removing code from `core` until there is dependency outside of core. the idea is to come to a stage where we can just delete the core folder entirely.